src/create-glamorous.js

```diff
@@ -101,17 +104,22 @@ export default function createGlamorous(splitProps) {
             typeof GlamorousComponent.comp === 'function' &&
             !GlamorousComponent.comp.prototype.render
 
-          return React.createElement(GlamorousComponent.comp, {
-            ...toForward,
-            ref: isStatelessFunction ? undefined : this.onRef,
-            style: fullStyles.length > 0 ? fullStyles : null,
-          })
+          return React.createElement(
+            GlamorousComponent.comp,
+            {
+              ...toForward,
+              ref: isStatelessFunction ? undefined : this.onRef,
+              style: fullStyles.length > 0 ? fullStyles : null,
+            },
+            ...spreadableChildren,
```

Comment:
[From the React docs](https://facebook.github.io/react/docs/react-api.html#createelement), while it does look like `children` is variadic, what happens if we just pass children as an array here?

Reply:
I just tried passing `children`, without wrapping as an array here, and things seemed to work as well!
![simulator screen shot - iphone 8 - 2017-09-22 at 17 16 04](https://user-images.githubusercontent.com/1051453/30764605-beae3c5a-9fb9-11e7-9908-475fcdbe0d8c.png)

Reply:
To be perfectly honest - I have no idea.. I just know the indented interface is for multiple children to be passed in as each argument. A little nervous to try otherwise without some additional research

Reply:
FWIW `styled-components` appears to just [pass the `children` through as the third argument](https://github.com/styled-components/styled-components/blob/ec8bb67e067d8e1187cca6fdd9468bfcc93cd364/src/models/StyledNativeComponent.js#L163)

Reply:
Another alternative here could be to hide behind JSX, since we know that syntax a bit better?

Reply:
My vote is to just pass `children` through as is.

Reply:
Seeing other usages where `children` is passed right through.. and the fact that your example below worked fine -- I'm down to pass it in directly 👍 

I also recall that `children` is a `PropTypes.node`, which arrays are valid types for. So this should be fine!